METADATA_SUPPORT.md

```diff
@@ -4,543 +4,542 @@ This list compares metadata types found in Salesforce v55 with the [metadata reg
+| AccountingSettings                    | ✅      |                                                                  |
+| BotBlock                              | ❌      | Not supported, but support could be added                        |
+| BotBlockVersion                       | ❌      | Not supported, but support could be added                        |
+| CollectionsDashboardSettings          | ✅      |                                                                  |
+| CustomizablePropensityScoringSettings | ✅      |                                                                  |
+| DataPackageKitDefinition              | ✅      |                                                                  |
+| DataPackageKitObject                  | ✅      |                                                                  |
+| DataSourceBundleDefinition            | ✅      |                                                                  |
+| DataSrcDataModelFieldMap              | ✅      |                                                                  |
+| DataStreamTemplate                    | ✅      |                                                                  |
+| DigitalExperience                     | ❌      | Not supported, but support could be added (but not for tracking) |
+| DigitalExperienceBundle               | ❌      | Not supported, but support could be added (but not for tracking) |
+| DigitalExperienceConfig               | ❌      | Not supported, but support could be added (but not for tracking) |
+| ExplainabilityMsgTemplate             | ❌      | Not supported, but support could be added                        |
+| ExpressionSetObjectAlias              | ❌      | Not supported, but support could be added                        |
+| FuelType                              | ❌      | Not supported, but support could be added                        |
+| FuelTypeSustnUom                      | ❌      | Not supported, but support could be added                        |
+| IncludeEstTaxInQuoteSettings          | ✅      |                                                                  |
+| MfgServiceConsoleSettings             | ✅      |                                                                  |
+| OauthOidcSettings                     | ✅      |                                                                  |
+| PortalDelegablePermissionSet          | ❌      | Not supported, but support could be added                        |
+| ReportingTypeConfig                   | ❌      | Not supported, but support could be added                        |
+| SustainabilityUom                     | ❌      | Not supported, but support could be added                        |
+| SustnUomConversion                    | ❌      | Not supported, but support could be added                        |
+| UserAccessPolicy                      | ❌      | Not supported, but support could be added                        |
```

Comment:
Sorry I'm out of the loop on this one.  Why did the format change so much here?

Reply:
Assuming the format change was intentional and correct we're good to go 👍

Reply:
I don't know, I'll have a look